deploy/platform/docker/config/rabbitmq/otel-collector-config.yaml

```diff
@@ -0,0 +1,80 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+receivers:
+  prometheus:
+    config:
+      scrape_configs:
+        - job_name: 'rabbitmq-monitoring'
+          scrape_interval: 30s
+          static_configs:
+            - targets:
+                - rmq0:15692
+                - rmq1:15692
+                - rmq2:15692
```

Comment:
This looks too overwhelming for a demo 😄

Reply:
> This looks too overwhelming for a demo 😄 I mean 3 nodes

The demo environment actually takes effect through the deploy.kubernetes. The demo deployed on deploy.kubernetes has just only one node.

Reply:
If this doesn't work, let's remove them to avoid confusion.

Reply:
> If this doesn't work, let's remove them to avoid confusion.

These are just two deployment methods examples (docker & k8s). Since our demo environment is deployed using `deploy.kubernetes`, I have limited the number of nodes to save resources.

Reply:
I mean, for both k8s and docker, is there any case we are going to use 3 nodes?

Reply:
Both of these methods can support multiple nodes, but for the purpose of our real  deploy demo env, I believe there is no need for three nodes. One node is sufficient to achieve the purpose of the showcase.

Reply:
Then why do we put 3 nodes(for example) here? I think `- rmq0:15692` has been showing this is a list. Anyone knows about rabbitMQ and k8s, they should be able to add more nodes if they need to, right?
The point is only, let's remove any unnecessary thing from the codes.

Reply:
ohhhh~~~,  I seem to know your point.